src/components/select/select.ios.scss

```diff
@@ -54,3 +58,8 @@ $select-ios-placeholder-color:   $select-ios-icon-color !default;
 
   pointer-events: none;
 }
+
+[dir="rtl"] .select-ios .select-icon .select-icon-inner {
+  right: 5px;
+  left: auto;
+}
```

Comment:
the lint rules for ionic indicate a need for a new line at the end of the file.
You can run it locally, `gulp lint.sass`, but you need to install `ruby` for that